pages/about.js

```diff
@@ -109,4 +116,31 @@ about.defaultProps = {
       titles: ['Associate Professor', 'Ryerson University'],
     },
   },
+  researchers: {
+    one: {
+      name: 'Julia Gervaslo',
+      titles: ['MA student, Ryerson University'],
+    },
+  },
+  techSupport: {
+    one: {
+      name: 'Likeable IT Inc',
+      titles: [],
+    },
+    two: {
```

Comment:
i wanted to mention this yesterday but that is the reason to do `one:` , `two:` , instead of like an array?

Reply:
Yeah I think using arrays would be ideal.
https://github.com/Cache-and-Cookies/msmr/pull/52/files#diff-8840d57d34462c5f7a759f74ca63759a6ab4fc1f46d91d9ee2239df35b119f91R14
Then you can just use `map` directly without doing `Object.keys()`

Reply:
the array data structure seems to better represent this relationship

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
yeah thanks for bringing it up